wriologin.js

```diff
@@ -0,0 +1,85 @@
+"use strict";
```

Comment:
use loginWithSessionId function to auth, example can be seen in Titter or Storage or webgold application. Skype me if any questions.